packages/unity-bootstrap-theme/stories/molecules/tabbed-panels/tabbed-panels.js

```diff
@@ -1,94 +1,128 @@
     }
-  });
+    if (scrollPosition < navItems.length - 1 && direction == -1) {
+      scrollPosition += 1;
+    }
+    parentNav[0].dataset.scrollPosition = scrollPosition;
 
-  const setButtonsCompatibility = (e) => {
-    const targets = ['a', 'button'];
-    if (targets.includes(e.target.localName)) {
-      e.target.focus();
+    scrollOffset = 0;
+    for (var i = 0; i < scrollPosition; i++) {
+      scrollOffset +=
+        navItems[i].offsetWidth +
+        parseInt(getComputedStyle(navItems[i]).marginLeft, 10) +
+        parseInt(getComputedStyle(navItems[i]).marginRight, 10);
     }
+
+    // set the position of the scroll of the .nav-tabs element
+    parentNav[0].scrollLeft = scrollOffset;
+    setControlVisibility(clicked, scrollOffset);
   };
 
   const setControlVisibility = (clicked, scrollOffset) => {
-    const parentContainer = $(clicked).closest('.uds-tabbed-panels');
-    const parentNav = $(clicked).siblings('.nav-tabs');
-    const scrollPosition = parentNav.data('scroll-position') * 1;
-    const tabPosition = parentNav[0].scrollWidth - scrollOffset;
+    // seleccionamos el ancestro mas cercano que tenga la clase .uds-tabbed-panels
+    const parentContainer = clicked.closest(".uds-tabbed-panels");
+    // seleccionamos los elementos padres hermanos del elemento clickeado
+    const parentNav = parentContainer.querySelector(".nav-tabs");
+
+    // obtenemos el valor del atributo data-scroll-position y nos asguaramos que sea un número entero
+    const scrollPosition = parseInt(parentNav.dataset.scrollPosition, 10);
+    const tabPosition = parentNav.scrollWidth - scrollOffset;
 
+    // ocultamos o mostramos los botones de scroll en función de la posición del scroll
     if (scrollPosition == 0) {
-      parentContainer.find('.scroll-control-prev').hide();
+      parentContainer.querySelector(".scroll-control-prev").style.display =
+        "none";
     } else {
-      parentContainer.find('.scroll-control-prev').show();
+      parentContainer.querySelector(".scroll-control-prev").style.display =
+        "block";
     }
-    if (tabPosition <= parentContainer.width()) {
-      parentContainer.find('.scroll-control-next').hide();
+    if (tabPosition <= parentContainer.offsetWidth) {
+      parentContainer.querySelector(".scroll-control-next").style.display =
+        "none";
     } else {
-      parentContainer.find('.scroll-control-next').show();
+      parentContainer.querySelector(".scroll-control-next").style.display =
+        "block";
     }
   };
 
-  const slideNav = (clicked, e, direction) => {
-    e.preventDefault();
-    const parentNav = $(clicked).siblings('.nav-tabs');
-    let scrollPosition = parentNav.data('scroll-position') * 1;
-    const navItems = parentNav.find('.nav-item').toArray();
-    let scrollOffset = parentNav.css('left').replace('px', '') * 1;
-    var adjustNavItem = 0;
+  // wait to load the page and all resources before initializing
+  window.addEventListener("load", () => {
```

Comment:
Inconsistent with other files where you declared const variables for  selectors and events ex: const DOM_ELEMENT_VIDEO = "video";